backend/dal/Migrations/v01.12.02/Up/PreUp/01-AdministrativeAreas.sql

```diff
@@ -0,0 +1,9 @@
+PRINT 'Removing duplicates'
+
+DELETE FROM dbo.[AdministrativeAreas]
+WHERE [Id] NOT IN (SELECT MAX(Id)
```

Comment:
If read it correctly, you're grouping by name and returning the max ID.  So if there are two of the items it will delete the first one.